parm/config/config.landanl

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -x
+
+########## config.landanl ##########
+# configuration common to all land analysis tasks
+
+echo "BEGIN: config.landanl"
+
+obs_list_name=gdas_land_adpsfc_only.yaml
+if [[ "${cyc}" == "18" ]]; then
+    obs_list_name=gdas_land_prototype.yaml
+fi
+
+export OBS_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/land/obs/config/
+export OBS_LIST=${HOMEgfs}/sorc/gdas.cd/parm/land/obs/lists/${obs_list_name}
+export LANDVARYAML=${HOMEgfs}/sorc/gdas.cd/parm/land/letkfoi/letkfoi.yaml
+export FV3JEDI_FIX=${HOMEgfs}/fix/gdas
+
+export io_layout_x=@IO_LAYOUT_X@
+export io_layout_y=@IO_LAYOUT_Y@
+
+export JEDIVAREXE=${HOMEgfs}/exec/fv3jedi_letkf.x
```

Comment:
It is not a "variational executable", if running an LETKF, is it?
Also, I think @RussTreadon-NOAA updated this to `JEDIEXE`.

Reply:
I made the changes as suggested and updated this to JEDIEXE.